dfencoder/scalers.py

```diff
@@ -48,6 +44,48 @@ def fit_transform(self, x: torch.Tensor):
         self.fit(x)
         return self.transform(x)
 
+class ModifiedScaler(object):
+    """Implements scaling using modified z score."""
+
+    def __init__(self):
+        self.median = None
+        self.mad = None # median absolute deviation
+        self.meanad = None # mean absolute deviation
```

Comment:
If you initialize these with `None`, add type annotations to specify what type they are.